src/components/ads/ad-utils/pbhAdFunctions.js

```diff
@@ -0,0 +1,70 @@
+import { AD_TYPES, AD_FUNCS_MAP } from "../ad-constants";
+import { getCreateAdFuncForType } from "./getCreateAdFuncForType";
+import { getAdId } from "./getAdId";
+
+/**
+ * getAdRegistrationID
+ *
+ * Registers instance of PBH ad function for given ad type and returns the
+ * associated ID. Used in the MakeAdComponent HOC.
+ *
+ * @param {String} adType - the type of add from the AD_TYPE enum in constants
+ * @returns {String} id
+ */
+export const getAdRegistrationID = adType => {
+  const adFunc = getCreateAdFuncForType(adType);
+
+  if (!adFunc) return;
+
+  const adId = getAdId();
+
+  registerAdFunctionWithId(adFunc, adId);
+
+  return adId;
+};
+
+/**
+ * startAds
+ *
+ * Appends pbh_start_ads to the end of the PbhAdUnit_load queue array.
+ * Used in the App component's componentDidMount method
+ */
+export const startAds = () => {
+  getAdLoadArray().push(() => window.pbh_start_ads());
```

Comment:
(minor) if you are wrapping the arrays anyway, what do you think about wrapping the push as well, so it's just something like addLoadCallback(fn) and addInitCallback(fn)